engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -39,6 +39,7 @@ class EngineJobExecutionActor(replyTo: ActorRef,
                               jobStoreActor: ActorRef,
                               callCacheReadActor: ActorRef,
                               jobTokenDispenserActor: ActorRef,
+                              backendSingletonActor: Option[ActorRef],
```

Comment:
Very happy to hear alternatives? (perhaps something so awful we're certain to change it...? Pinging @mcovarr 😛)